pkg/filesystem/subvolume.go

```diff
@@ -52,6 +58,64 @@ func List(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace
 	listCephFSSubvolumes(ctx, clientsets, operatorNamespace, clusterNamespace, includeStaleOnly, subvolumeNames)
 }
 
+// checkForExternalStorage checks if the external mode is enabled.
+func checkForExternalStorage(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace string) bool {
+	enable := false
+	resource := "storageclusters"
```

Comment:
```suggestion
	resource := "storageclusters"
```
in upstream it is `cephClusters` not `stroageClusters`